### PR TITLE
Add includeBlunder argument (solves #16)

### PR DESCRIPTION
**Pull request by karol-brejna-i**
My proposal for dealing with omitting the blunder move (initial move of the puzzle) --solves #16 .

1. Puzzle generation stays untouched
2. Blunder move removal happens when exporting the puzzle to PGN
3. Move exporting a puzzle to PGN to a dedicated class (PgnExporter).

I am wondering if we shouldn't invert the logic (meaning) of the argument: having something like `skipBlunder` instead of `includeBlunder`.  Then we could have simpler construction, like `./main.py --skipBlunder`

### CODE REVIEW

**Comment by karol-brejna-i** [comment]
Some additional changes sneaked in:
1. Silencing the UCI engine (logging.getLogger("chess._engine").setLevel(logging.WARNING))

File: opening_counter.py [hunk 1]
```diff
@@ -8,20 +8,8 @@
 import chess.pgn
 
 parser = argparse.ArgumentParser(description=__doc__)
```
Comment (vitogit) [inline opening_counter.py:10]: You can delete this file,  it was an experiment and I don't think it's add value to the project.

**Review by vitogit — APPROVED** [review]
Excellent work. thanks!

**Comment by vitogit** [comment]
> I am wondering if we shouldn't invert the logic (meaning) of the argument: having something like `skipBlunder` instead of `includeBlunder`. Then we could have simpler construction, like `./main.py --skipBlunder`
I think we can have shorthands for the params, like `-sb` to skip blunder or just `-s`, `-q` for the quiet param, etc

File: opening_counter.py (hunk 1 above)
Comment (karol-brejna-i) [reply opening_counter.py]: Done

**Comment by vitogit** [comment]
I opened a ticket in case we want to change how the arguments works https://github.com/vitogit/pgn-tactics-generator/issues/22

**Pull request merged**